core/src/test/java/org/pentaho/platform/scheduler2/action/SchedulerFilenameUtilsTest.java

```diff
@@ -83,8 +86,20 @@ public void testConcat_Repository() {
       SchedulerFilenameUtils.concat( "/home/randomUser",
         "/afolder" )
     );
+
+    // TEST pathToAdd is many folders + filename
```

Comment:
Added new test cases to demonstrate fully supported `SchedulerFilenameUtils#concat`